pallet/src/tests.rs

```diff
@@ -1,7 +1,57 @@
 #![cfg(test)]
 
-use crate::mock::*;
+use crate::{migrations, mock::*, pallet};
 use frame_support::assert_ok;
+use frame_support::traits::{GetStorageVersion, StorageVersion};
+
+#[test]
+fn migration_from_v0_to_v1_works() {
```

Comment:
should we also test migration when there is no validators change pending?

Reply:
In this case we would probbaly be fine not running any migration at all (or en empty on that just updates the version) - the storage layer automatically removes the storage entry if the value is None, so next time sth is stored there it's stored with a correct schema.

Reply:
I guess so, but for me this is still a scenario worth exposing in tests. Up to you, a non-blocker